### DISCUSSION TITLE
redis_dns.rdb 这个文件要多久才能生成啊，用了半天了没发现这个文件

### DISCUSSION THREAD

**Discussion by mrzj110** [kkkgo/PaoPaoDNS#12] [问答和帮助]
redis_dns.rdb 这个文件要多久才能生成啊，用了半天了没发现这个文件

**Comment by kkkgo** [comment]
当数据量达到redis的最低转储体积就会生成，按配置最低2小时后才会转储一次，你可以观察一下。

**Comment by mrzj110** [comment]
> 当数据量达到redis的最低转储量就会生成，按配置最低2小时后才会转储一次，你可以查看一下。

好的，谢谢

**Comment by Lovest20018** [comment]
docker run --rm -it sliamb/paopao-pref -server 10.10.10.8 -pc 68 -limit 50 -sleep 0ms -timeout 1s
用这条参数跑的时候rdb大小不会变化吗？只能等全部跑完才可以吗？

**Reply by kkkgo** [reply]
最低2小时后才会转储一次

**Comment by Lovest20018** [comment]
@kkkgo  大佬，对于ipv6实在迷糊，看您之前也是用tp的路由器，tp的话，应该是用
![1111](https://github.com/kkkgo/PaoPaoDNS/assets/30313973/e5c10bc0-fc60-4277-a38c-a353c8853c83)
这里设置吧，那这里的dns应该填ppdns的ipv6地址吧？
还是为空用运营商的dns呢？如果用ppdns的ipv6地址，直接填linux的公网地址吗？这里实在迷糊？公网地址应该会变的吧？
这方面实在是不懂，十分感谢这两天您的回复。。

**Reply by kkkgo** [reply]
如果能开ssh的话进去配置文件把IPV6 DNS关掉，我是这么干的。不能改就填一个不存在的IPV6地址。如果有问题可以另开贴。

**Reply by Lovest20018** [reply]
恩在您的blog上看到了，晚上对着您的教程改下。感谢。我这款5408M应该跟您之前的固件一样。看到有诊断模式。
